packages/storage/Cargo.toml

```diff
@@ -10,10 +10,11 @@ license = "Apache-2.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
+default = ["iterator"]
 # This enables iterator functionality, as exposed in cosmwasm-std/iterator
 iterator = ["cosmwasm-std/iterator"]
 
 [dependencies]
 # Uses the path when built locally; uses the given version from crates.io when published
-cosmwasm-std = { path = "../std", version = "0.14.0-0.4.0" }
+cosmwasm-std = { path = "../std", version = "0.14.0-0.4.0", default-features = false }
```

Comment:
What is the mean `deafult-features=false`?

Reply:
This comes from https://github.com/CosmWasm/cosmwasm/pull/993 and vm and storage do not need std's iterator featuer.